### DISCUSSION TITLE
Possible to force color output on CircleCI orb?

### DISCUSSION THREAD

**Discussion by andyshinn** [Checkmarx/kics#5383] [Q&A] [answered]
KICS isn't outputting color on CircleCI jobs. I don't see any color options in the orb settings. Possible to support color output on CircleCI?

**Comment by rafaela-soares** [comment, marked as answer]
Hello, @andyshinn!

Thank you so much for another great suggestion! 🚀 We are working on it!

**Explanation**: CircleCI jobs do not display the color output because the KICS Dockerfile is not supporting it. However, during the tests of the color support in the KICS Dockerfile (only in deployment), it was noticed that Circle CI presents a different behavior from the docker run. We contacted Circle CI support to understand what we are missing. For example:

- Expected
![image](https://user-images.githubusercontent.com/74001161/171036171-5d7f1e65-616e-4f59-8080-e5048c5996d6.png)

- Circle CI
![image](https://user-images.githubusercontent.com/74001161/171036181-29b12264-be53-411d-9e3c-9fcb638e7a74.png)

We will keep you updated about the progress status 😊

**Reply by rafaela-soares** [reply]
Hello again, @andyshinn! 

We opened a new feature request in CircleCI Ideas to have the console in CircleCI match the usual docker run output. If you could upvote the [feature request](https://circleci.canny.io/feature-requests/p/console-in-circleci-to-match-the-usual-docker-run-output), it will be great 😊
